scripts/java-api-check.json

```diff
@@ -10,7 +10,55 @@
       "filter": {
         "classes": {
           "regex": true,
-          "include": ["com\\.mapbox\\.maps\\..*"]
+          "include": ["com\\.mapbox\\.maps\\..*"],
+          "exclude": [
+            "com\\.mapbox\\.maps\\.NativeMapImpl",
```

Comment:
maybe add some script and compare this list and new?
stop run/publish if find some differences

Reply:
Do we really want to exclude these internal classes and only care about Kotlin compatibility? as these are public to Java users.

Reply:
@pengdev If we need to consider this, we can't change these internal classes before v11.

Reply:
@alexander-kulikovskii the api check is just a reminder for breaking changes, it will not block any process.
`revapi` treats internal classes changes as breaking change, this pr excludes these internal classes to reduce the warning message for these internal classes.

Reply:
@Chaoba I'm ok with skipping checking these internal classes for now to make the revapi checker more usable.
I think we should treat the Java API surface for Kotlin internal classes as a tech debt that we should address in v11. Let's ticket it out to keep track and address in the next major release.